### PR TITLE
updated anchors in links

### PR DESCRIPTION
**Pull request by jc-berger**
See Doc team's Jira: https://issues.redhat.com/browse/ACM-9884 

See Gdoc where Kevin Cormier and I collaborated: https://docs.google.com/document/d/1KomyPBRsnPcS2g-HtgRDdyrXWiuzqBl-h9A-5eOB5jc/edit?usp=sharing

### CODE REVIEW

**Comment by swopebe** [comment]
Amazing. Looks like it was not too much to update this release, so I am thankful for that!

File: frontend/src/lib/doc-util.tsx [hunk 1]
```diff
@@ -55,9 +55,9 @@ export const DOC_LINKS = {
   HYPERSHIFT_INTRO: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-intro`,
   HYPERSHIFT_DEPLOY_AWS: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-aws`,
   HYPERSHIFT_MANAGE_KUBEVIRT: `${DOC_BASE_PATH}/clusters/index#hosted-control-planes-manage-kubevirt`,
-  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#hosted-enable-feature-aws`,
+  HOSTED_ENABLE_FEATURE_AWS: `${DOC_BASE_PATH}/clusters/index#hosting-service-cluster-configure-aws`,
```
Comment (KevinFCormier) [inline frontend/src/lib/doc-util.tsx:58]: @fxiang1 The doc anchor we were using only existed until ACM 2.8; can you confirm if this is the best one to use now?

Comment (fxiang1) [reply frontend/src/lib/doc-util.tsx:58]: @KevinFCormier  Good catch! Looks like it should be this one now:
https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html-single/clusters/index#enable-or-disable-hosted-control-planes

Can you change the anchor to this?
#enable-or-disable-hosted-control-planes

Comment (KevinFCormier) [reply frontend/src/lib/doc-util.tsx]: No problem. Updated.

Comment (fxiang1) [reply frontend/src/lib/doc-util.tsx]: Thanks!

**Comment by jc-berger** [comment]
@KevinFCormier and @fxiang1 thanks for your reviews and for implementing changes! Let me know if you think this PR can be merged. Otherwise, I'm happy to make more changes as you see fit.

**Review by fxiang1 — APPROVED** [review]
Thanks Kevin for fixing all the outdated links!

**Pull request merged**